lib/topological_inventory/openshift/operations/worker.rb

```diff
@@ -36,37 +35,33 @@ def stop
 
         private
 
-        attr_accessor :messaging_client_opts, :client, :api_client, :sleep_poll, :poll_timeout
+        attr_accessor :messaging_client_opts, :client, :sleep_poll, :poll_timeout
 
         def process_message(client, msg)
           logger.info("Processing #{msg.message} with msg: #{msg.payload}")
           # TODO: Move to separate module later when more message types are expected aside from just ordering
-          order_service(client, msg)
+          order_service(msg.payload)
+          client.ack(msg.ack_ref)
         rescue StandardError => e
           logger.error(e.message)
           logger.error(e.backtrace.join("\n"))
           nil
         end
 
-        def order_service(client, msg)
-          task_id, service_plan_id, order_params = msg.payload.values_at("task_id", "service_plan_id", "order_params")
+        def order_service(payload)
+          task_id, service_plan_id, order_params = payload.values_at("task_id", "service_plan_id", "order_params")
 
           service_plan     = api_client.show_service_plan(service_plan_id)
           service_offering = api_client.show_service_offering(service_plan.service_offering_id)
+          source_id        = service_plan.source_id
 
-          catalog_client = Core::ServiceCatalogClient.new(service_plan.source_id)
+          catalog_client = Core::ServiceCatalogClient.new(source_id)
 
-          logger.info("Ordering #{service_offering.name} #{service_plan.name}...")
```

Comment:
Did we lose the logging?

Reply:
Oh yeah forgot to add that back after I stopped passing around the service_offiering and service_plan objects

Reply:
Done